modules/aws-installing-an-aws-load-balancer-operator.adoc

```diff
@@ -0,0 +1,373 @@
+// Module included in the following assemblies:
+//
+// * networking/aws-load-balancer-operator.adoc
+
+:_content-type: PROCEDURE
+[id="aws-installing-an-aws-load-balancer-operator_{context}"]
+= Installing an AWS Load Balancer Operator
+You can install an AWS Load Balancer Operator (ALBO) if you meet certain requirements.
+
+.Prerequisites
+
+* You have an existing
+ifndef::openshift-rosa[]
+{product-title}
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+{product-title} (ROSA)
+endif::openshift-rosa[]
+cluster with bring-your-own-VPC (BYO-VPC) configuration across multiple availability zones (AZ) installed in STS mode.
+* You have access to the cluster as a user with the `dedicated-admin` role.
```

Comment:
Should there be a non-ROSA step to have `cluster-admin` access?

Reply:
I was told by SRE that we should not include `cluster-admin` access in the doc.